src/js/gi/containers/GiBillApp.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Modals from '../containers/Modals';
+import PreviewBanner from '../components/PreviewBanner';
+import Breadcrumbs from '../components/Breadcrumbs';
+import AboutThisTool from '../components/AboutThisTool';
+
+function AppContent({ children, isDataAvailable }) {
```

Comment:
If veterans do not need to be registered to use this tool, we can get rid of the `AppContent` wrapper. It is meant to be used in conjunction with a `RequiredLoginView` component that wraps the entire thing (and calls AppContent after the user logs in successfully).